hypothesis-python/src/hypothesis/internal/conjecture/utils.py

```diff
@@ -167,7 +167,7 @@ def boolean(data):
 
 
 def biased_coin(data, p):
-    """Return False with probability p (assuming a uniform generator),
+    """Return True with probability p (assuming a uniform generator),
```

Comment:
Probably worth putting this in a separate PR so it's easier for David to review; sorry 😞

Reply:
I wrote a separate commit that reverts this - unsure if that is sufficient